planner/core/exhaust_physical_plans.go

```diff
@@ -1470,11 +1474,26 @@ func (p *LogicalJoin) exhaustPhysicalPlans(prop *property.PhysicalProperty) ([]P
 		}
 	})
 
+	if prop.IsFlashOnlyProp() && ((p.preferJoinType&preferBCJoin) == 0 && p.preferJoinType > 0) {
+		return nil, false
+	}
+	joins := make([]PhysicalPlan, 0, 5)
+	if p.ctx.GetSessionVars().AllowBCJ {
+		broadCastJoins := p.tryToGetBroadCastJoin(prop)
+		if (p.preferJoinType & preferBCJoin) > 0 {
+			logutil.BgLogger().Info("prefer bc join", zap.Int("bc count", len(broadCastJoins)))
```

Comment:
just for test? bc -> broadcast maybe better.

Reply:
Ok, this log is verbose, I will remove it